Zend/zend_API.c

```diff
@@ -2784,7 +2784,7 @@ static int zend_is_callable_check_class(zend_string *name, zend_class_entry *sco
 	*strict_class = 0;
 	if (zend_string_equals_literal(lcname, "self")) {
 		if (!scope) {
-			if (error) *error = estrdup("cannot access self:: when no class scope is active");
+			if (error) *error = estrdup("cannot access self when no class scope is active");
```

Comment:
In `zend_ast.c` we use `Cannot use "self" when no class scope is active`. Shouldn't these messages be consolidated into one? What should be the preferred format?